test/lish/symlinks_test.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:tar/tar.dart';
+import 'package:test/test.dart';
+
+import '../descriptor.dart';
```

Comment:
```suggestion
import '../descriptor.dart' as d;
```
To follow the pattern we have in other files.

Reply:
Done